### ISSUE TITLE
Button to cancel computation

### ISSUE THREAD

**Issue by Acefall** [data-hydenv/hydenv-database#13]
This is a nice to have.
On my machine some queries, especially when plotting the countries of the world, take 30 seconds to execute. During this computation the app is non responsive.
A "cancel/stop execution" button would be nice.

**Comment by mmaelicke** [comment]
Yeah, thought about that. Unfortunately, this is not really possible like this, as stopping a SQL query on postgres needs elevated privileges. Thus I'll put a won't fix label on this...
